stablehlo/integrations/c/CMakeLists.txt

```diff
@@ -28,7 +28,6 @@ add_mlir_public_c_api_library(ChloCAPI
 
   LINK_LIBS PUBLIC
   ChloOps
-  LLVMSupport
```

Comment:
Can we add a comment `# LLVMSupport def handled in top level CMake`

Here and in other file. Otherwise I think there's a decent chance it gets added back

Reply:
done!